### PR TITLE
Add democracyOS, Madison, brigade hub

### PR DESCRIPTION
**Pull request by carlvlewis**
Added DemocracyOS and Madison -- both tools for participatory governance.
Add BrigadeHub -- An all-in-one suite and CMS for meetup websites.

### CODE REVIEW

**Comment by carlvlewis** [comment]
Add three tools to open source.

File: README.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ Below is a curated list of awesome digital tools for activism. These are tools f
  * [TakeCharge.io](https://github.com/controlshift/prague-server) - opensource or SaaS online donations powered by Stripe.
  * [VictoryKit](http://www.victorykitapp.com) - Optimize petition campaign content. Not actively maintained.
  * [Countdown](https://github.com/drewrwilson/countdown) - A quick & simple tool for making your own webpage with a countdown to your upcoming event.
+ * [DemocracyOS](http://democracyos.org) - An open-source platform for public policy engagement, open feedback, and participatory governance.
```
Comment (drewrwilson) [inline README.md]: I just tried it out and looks like there's a heroku error on this site :(

Is this a project you're working on? If so, let me know when you get it back up again and I'll review it again. 

In the mean time I'll look at your other two suggestions

image

File: README.md [hunk 2]
```diff
@@ -28,6 +28,9 @@ Below is a curated list of awesome digital tools for activism. These are tools f
  * [TakeCharge.io](https://github.com/controlshift/prague-server) - opensource or SaaS online donations powered by Stripe.
  * [VictoryKit](http://www.victorykitapp.com) - Optimize petition campaign content. Not actively maintained.
  * [Countdown](https://github.com/drewrwilson/countdown) - A quick & simple tool for making your own webpage with a countdown to your upcoming event.
+ * [DemocracyOS](http://democracyos.org) - An open-source platform for public policy engagement, open feedback, and participatory governance. 
+ * [Madison](http://mymadison.io) - An open-source platform for policy proposals and petitions to be uploaded for robust public feedback and exchange.
```
Comment (drewrwilson) [inline README.md]: Looks like this is now retired. Looks awesome tho! 

image

File: README.md [hunk 3]
```diff
@@ -28,6 +28,9 @@ Below is a curated list of awesome digital tools for activism. These are tools f
...
+ * [BrigadeHub](https://brigadehub.github.io) - A full-fledged Node CMS with single-click deployment that allows you to focus on building your community, not on building your website. Integrates with Meetup.com API for events, users; Medium for blogging capabilities; and Slack.
```
Comment (drewrwilson) [inline README.md]: this one is no longer active any more either but, again, another really awesome project. 

image

**Comment by drewrwilson** [comment]
hey @carlvlewis! thank you for suggesting these. they are all AWESOME. But they're outdated :( so I'm going to close this pull request, but if you encounter any more cool tools please do definitely do more pull requests!

**Pull request closed without merging**